suite/sanity/loss_recover.go

```diff
@@ -26,11 +26,11 @@ const (
 type lossAndRecoveryParam struct {
 	installParam
 	// ReplaceNodeType : see killXXX constants
-	ReplaceNodeType string `json:"kill" validate:"required,eq=apimaster|clmaster|clbackup|worker"`
+	ReplaceNodeType string `json:"kill" validate:"required,oneof=apimaster clmaster clbackup worker"`
```

Comment:
This shows up several other places in the codebase that could probably be converted to the cleaner `oneof` syntax.  Theeothers aren't broken due to use of repeat `eq=`.

```
walt@work:~/git/robotest$ rg 'validate.*eq.*\|'                                                                                                                                         [141]
infra/gravity/config.go
72:	CloudProvider string `yaml:"cloud" validate:"required,eq=aws|eq=azure|eq=gce|eq=ops"`

infra/terraform/config.go
75:	CloudProvider string `validate:"required,eq=aws|eq=azure|eq=gce"`
83:	OS string `json:"os" yaml:"os" validate:"required,eq=ubuntu|eq=redhat|eq=centos|eq=debian|eq=suse"`

e2e/framework/test_context.go
167:	CloudProvider string `json:"cloud_provider" yaml:"cloud_provider" validate:"omitempty,eq=aws|eq=azure"`
286:	Type defaults.ClusterAddressType `json:"type" yaml:"type" validate:"eq=direct|eq=loadbalancer|eq=public"`
309:	OS string `json:"os" yaml:"os" validate:"required,eq=ubuntu|eq=redhat|eq=centos|eq=debian"`
```